example/particle_tracers/particle_tracers.cpp

```diff
@@ -265,8 +265,7 @@ TaskStatus CalculateFluxes(MeshBlockData<Real> *mbd) {
 
   auto advected = mbd->Get("advected").data;
 
-  auto x1flux = mbd->Get("advected").flux[X1DIR].Get<4>();
-
+  parthenon::ParArray4D<Real> x1flux = mbd->Get("bnd_flux::advected").data.Get(1, 0, 0);
```

Comment:
Yes, it is disturbing that this test passes with that bug. I am not really sure what it is supposed to be doing though... 

The `Get` function called here is really `ParArrayGeneric::Get(Args... args)`, so I don't think we can reasonably make a flux based overload. I don't think we really need to support it from `MeshBlockData` either, since I don't think we want to encourage the pattern that is used here (variables should be accessed through packs rather than pulling out raw `ParArray`s directly from `MeshBlockData`). That being said, I agree that the `ParArrayGeneric::Get` interface is a bit confusing and should possibly be changed. That is for another time though I think...